spec/models/monitoring_review_spec.rb

```diff
@@ -8,12 +8,30 @@
     described_class.new id, access_token
   end
 
+  describe "#itams_url" do
+    it "returns a URL" do
+      expect(subject.start_date).to be_a(String)
+    end
+  end
+
+  describe "#review_type" do
+    it "returns the URL to view the report in IT-AMS" do
+      expect(subject.itams_url).to eq "https://example.com/TODO"
```

Comment:
[issue] this isn't testing the review_type!

Reply:
wow I really did a good job on this part